requirements.txt

```diff
@@ -9,6 +9,8 @@ moto
 mypy
 brotli
 brotli-asgi
+awslambdaric
+awslambdaric-stubs
```

Comment:
@emcpow2 
Thank you for creating this PR.
Did we need `awslambdaric` for type-checking?
I guess `awslambdaric-stubs` provides type stub.

```python
from mangum import Mangum

Mangum()({}, {})
```

```bash
$ mypy main.py                                                                                                                                     ✘ 1 
main.py:3: error: Missing positional argument "app" in call to "Mangum"
main.py:3: error: Argument 2 to "__call__" of "Mangum" has incompatible type "Dict[<nothing>, <nothing>]"; expected "LambdaContext"
Found 2 errors in 1 file (checked 1 source file)

$ pip list |grep aws                                                                                                                             
awslambdaric-stubs       0.1.0

```

Reply:
valid concern

removing redundant dependency